api/client/cluster/services.go

```diff
@@ -0,0 +1,74 @@
+package cluster
+
+import (
+	"github.com/libopenstorage/openstorage/services"
+)
+
+// ServiceEnterMaintenanceMode takes node out of cluster
+func (c *clusterClient) ServiceEnterMaintenanceMode(exitOut bool) error {
```

Comment:
What is exitOut? Probably not needed right?

Reply:
right, I commented about same above^